dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/CompanyDataController.kt

```diff
@@ -30,8 +33,13 @@ class CompanyDataController(
         onlyCompanyNames: Boolean
     ): ResponseEntity<List<StoredCompany>> {
         return if (selectedIndex == null) {
+            logger.info(
+                "Received a request to get companies with " +
+                    "searchString='$searchString', onlyCompanyNames='$onlyCompanyNames'"
+            )
             ResponseEntity.ok(dataManager.searchCompanies(searchString ?: "", onlyCompanyNames))
         } else {
+            logger.info("Received a request to get companies by stock index '$selectedIndex'")
             ResponseEntity.ok(dataManager.searchCompaniesByIndex(selectedIndex))
         }
     }
```

Comment:
(Kommentar gilt generell und hat nichts mit dem File hier zu tun)

Entsprechend unserer conversation im PR des datalandEDC-Repos sollten/könnten wir `--stacktrace` auch in diesem Repo hinter `./gradlew` Befehle packen.

Reply:
fixed

Reply:
Es gibt in dataland-backend ein docker file namens `DockerfileTest`, das beim Profil "testing" im `docker-compose.yml` gestartet wird.

Dieses führt einen CMD aus `CMD ./gradlew :dataland-backend:bootRun --no-daemon`.

Würde es Sinn machen, hier auch `--stacktrace` zu flaggen um mehr logs zu haben?

Reply:
passt jetzt